MBBSEmu/HostProcess/ExportedModules/Galgsbl.cs

```diff
@@ -995,5 +999,26 @@ private void btuinp()
             //Clear the Buffer
             channel.InputBuffer.SetLength(0);
         }
+
+        /// <summary>
+        ///     Takes the specified character and adds it directly to the channel input buffer
+        /// 
+        ///     Signature: void chiinp(int chan,char c);
+        /// </summary>
+        private void chiinp()
+        {
+            var channelNumber = GetParameter(0);
+            var character = (byte)GetParameter(1);
+
+            if (!ChannelDictionary.TryGetValue(channelNumber, out var channel))
+            {
+                Registers.AX = ERROR_CHANNEL_NOT_DEFINED;
+                return;
+            }
+
+            channel.InputBuffer.WriteByte(character);
+
+            _logger.Debug($"Adding {character:X2} to input buffer");
```

Comment:
not necessary? With the tests we should have confidence that this works, thus reducing the need to verify via logs?

Or at least guard behind `#ifdef DEBUG`